Regression in 2.8.1 – Cannot find module chrono-node/dist/src/common/abstractRefiners

Love this project (we [wrote a blog post](https://dub.co/blog/smart-datetime-picker) a while ago about how we're using it at Dub.co), thanks a lot for your work on this @wanasit 🙏 

Just a heads-up, there seems to be a regression in 2.8.1 that was released an hour ago – we're getting this error in [our E2E tests](https://github.com/dubinc/dub/actions/runs/15233350869/job/42843944802):

```
Error: Cannot find module '/home/runner/work/dub/dub/node_modules/.pnpm/chrono-node@2.8.1/node_modules/chrono-node/dist/src/common/abstractRefiners' imported from /home/runner/work/dub/dub/node_modules/.pnpm/chrono-node@2.8.1/node_modules/chrono-node/dist/esm/locales/ja/refiners/JPMergeWeekdayComponentRefiner.js
```

For now, we're pinning the 2.7.5 version to circumvent this issue, but I thought I'd create an issue here in case anyone else is running into the same error.

--- a/src/locales/ja/refiners/JPMergeWeekdayComponentRefiner.ts
+++ b/src/locales/ja/refiners/JPMergeWeekdayComponentRefiner.ts
@@ -1,4 +1,4 @@
-import { MergingRefiner } from "../../../../src/common/abstractRefiners";
+import { MergingRefiner } from "../../../common/abstractRefiners";
 import { ParsingResult } from "../../../results";
 
 /**

